docs/sources/panels/transformations/_index.md

```diff
@@ -29,7 +29,7 @@ Before you can configure and apply transformations:
 - [Add field from calculation]({{< relref "./types-options.md#add-field-from-calculation" >}})
 - [Concatenate fields]({{< relref "./types-options.md#concatenate-fields" >}})
 - [Config from query results]({{< relref "./config-from-query.md" >}})
-- [Convert field type]({{< relref "./convert-field-type.md" >}})
+- [Convert field type]({{< relref "./types-options.md#convert-field-type" >}})
```

Comment:
This change is unrelated? I think it might be just a small fix? Just confirming it's something that should be here and not an unwanted change.

Otherwise, LGTM!

Reply:
Ah yes, small fix that I ran into as I worked through the docs.